### DISCUSSION TITLE
System Status / Outage page?

### DISCUSSION THREAD

**Discussion by virtuallyvlad** [standardnotes/forum#2646] [Suggestions]
Is there a way to check on the sync server's status, current or past outages, and/or system load?

If there's nothing official, is there an unofficial way to do so?  Like by checking if [some URL](https://sync.standardnotes.org/) is working?

And if there is nothing, please allow this to serve as a feature request.  This would be a very helpful feature to have, especially in the app, to bring some peace of mind to users during outages and to avoid the inevitable threads every time there is some server issue.

**Comment by JaspalSuri** [comment]
Hi @virtuallyvlad, there currently isn't a system status page, but I will forward your suggestion to the dev team. We sometimes post the status of the servers on [twitter](http://twitter.com/standardnotes), though. 

Currently the main way to determine if the sync server is inaccessible is to simply edit or create a new note and see if you receive any error messages or if the changes that you made are being synced across your devices.

**Comment by virtuallyvlad** [comment]
> Hi @virtuallyvlad, there currently isn't a system status page, but I will forward your suggestion to the dev team. We sometimes post the status of the servers on [twitter](http://twitter.com/standardnotes), though.

Thanks.  I do check the twitter but, like you said, it's not always on there.  Searching for [twitter mentions](https://twitter.com/search?q=%40standardnotes) seems to be somewhat reliable, though, as people start complaining quickly.  :D

> Currently the main way to determine if the sync server is inaccessible is to simply edit or create a new note and see if you receive any error messages or if the changes that you made are being synced across your devices.

Yes that's how you'd usually find out but then there's always the question if it is that specific device or app, something with your network or connection, etc.

**EDIT**: Checking [SN's Slack](https://standardnotes.com/slack) may actually be the best/quickest way -- seems like the most active/responsive place for outage info

**Comment by virtuallyvlad** [comment]
FYI: as of 2024-04-28, the best place to check if there is an outage:
- **_[#sn-discussion channel](https://discord.com/channels/894600255058898974/894600255058898977) of the Discord server_**.

The following places had no mention of this current outage:
- https://discord.com/channels/894600255058898974 - Discord server; no other channels mentioned outages or sync issues.
- https://standardnotes.com/slack - Deprecated?  Now redirects to Discord.
- https://github.com/standardnotes/forum/discussions - No new threads mentioning outages or sync issues.
- https://github.com/standardnotes/forum/issues - No new threads mentioning outages or sync issues.
- https://twitter.com/search?q=%40standardnotes&f=live - No mention of outages or sync issues.
- https://twitter.com/standardnotes - No mention of outages or sync issues.
- https://sync.standardnotes.org/ - Nothing here; not a status page.
